pkg/csi_driver/node.go

```diff
@@ -331,6 +331,11 @@ func (s *nodeServer) NodeStageVolume(ctx context.Context, req *csi.NodeStageVolu
 		if unmntErr := mount.CleanupMountPoint(stagingTargetPath, s.mounter, false /* extensiveMountPointCheck */); unmntErr != nil {
 			klog.Errorf("Unmount %q failed: %v", stagingTargetPath, unmntErr.Error())
 		}
+		isVPCErr := strings.Contains(err.Error(), "exit status 32")
```

Comment:
32 may not be just for vpc errors

Reply:
Agreed, I should not aggressively make such assumption. So I'll simply pop a message to check VPC for now. Hopefully this can be solved once we improve static provisioned volumes as our H2 plan.

Reply:
Ack